### ISSUE TITLE
Footer Prop

### ISSUE THREAD

**Issue by ebarton** [lelandrichardson/react-native-parallax-view#27]
It would be great to have a Footer prop that allows for a parallax footer, revealed behind the scrollview or listview.

**Comment by jsmmth** [comment]
I agree! This would be nice to see. For example and what I find a common design practise is to have a location map at the end of a screen. It would be nice to allow this to grow as the header does when you scroll past the bottom!

**Comment by rajeshzmoke** [comment]
+1
